docs/root/start/quick-start/configuration-dynamic-filesystem.rst

```diff
@@ -8,6 +8,11 @@ protocol.
 
 When the files are changed on the filesystem, Envoy will automatically update its configuration.
 
+Notice:
+
+Now Enovy only get the changes when write to temporary file and rename into place, ignore overwritten directly.
+The purpose is  that atomic renames are the only way to achieve consistency.
+
```

Comment:
Thanks. May be a note `  .. note::` would be better.

Reply:
Sure, updated.